server/src/main/java/fr/uga/l3miage/tp4/models/CandidateEvaluationGridEntity.java

```diff
@@ -0,0 +1,27 @@
+package fr.uga.l3miage.tp4.models;
+
+import lombok.Getter;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+@Getter
+public class CandidateEvaluationGridEntity {
+    @Id
+    private Long sheetNumber;
+
+    private double grade;
+    private LocalDateTime submissionDate;
+
+    @ManyToOne
+    private CandidateEntity candidate;
+
+
+    @ManyToOne
+    private  ExamEntity exam;
```

Comment:
Ici le diagramme était pas dans le bon sens, mais une prise de recul est nécessaire, bizarre que des critère d'une évaluation porte l'eval elle même

```suggestion
    private  ExamEntity exam;
    
     @ManyToMany
    private Set<EvaluationCriteriaEntity> evaluationCriteriaEntities;
```